src/Tools/ExternalAccess/Razor/RazorCSharpInterceptionMiddleLayer.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Composition;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Host.Mef;
+using Microsoft.VisualStudio.LanguageServer.Client;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.Razor
+{
+    [Export(typeof(ILanguageClientMiddleLayer))]
```

Comment:
We'll want to export one of our own types instead of a platform type. Since we don't own the `ILanguageClientMiddleLayer` it's possible for anyone else in the ecosystem to also export it and cause chaos 😄